docs-api/frontity-plugins/embedded-mode.md

```diff
@@ -0,0 +1,69 @@
+# Frontity Embedded Mode
+
+This plugin enables an **Embedded Mode** implementation of Frontity. It can be downloaded from [here](https://github.com/frontity/frontity-embedded).
+
+Embedded Mode can be contrasted with **Decoupled Mode**. In Decoupled Mode the primary domain points to the Frontity site, with the WordPress site being on a secondary domain or on a subdomain of the primary domain.
+
+In Embedded Mode the primary domain points to the WordPress site, and the Frontity site can be on another domain including the default URL provided by a free hosting service such as [Vercel](https://docs.frontity.org/deployment/deploy-using-vercel).
+
+This plugin replaces the active WordPress theme with the Frontity installation. Frontity therefore effectively becomes the WordPress sites' theme. It works by substituting it's own [`template.php`](https://github.com/frontity/frontity-embedded/blob/master/includes/template.php) in place of any call made to the [WordPress template hierarchy](https://developer.wordpress.org/themes/basics/template-hierarchy/).
+
+## Advantages of Embedded Mode
+
+Embedded Mode offers several advantages over Decoupled Mode.
+
+- The WordPress site exists under the primary domain. This ensures that:
+  - the SEO of the site is not compromised
```

Comment:
What do you mean by this?

Reply:
I guess that should say 'any existing SEO for the site will not be compromised' - what do you think?

Reply:
Actually what I think I meant was that all the SEO benefits already built in to WordPress are still available.

Reply:
Ok, but as the title of this section is "Embedded mode offers several advantages over Decoupled mode" I think it implies that SEO is compromised in Decoupled mode.

Reply:
You're right - I'll look at rephrasing that.

Reply:
Done: https://github.com/frontity/api-reference/pull/34/commits/7cb15b98b19814873209110917d5fedfa98a1801